### ISSUE TITLE
Remote DroidGuard runtime for public use?

### ISSUE THREAD

**Issue by UjuiUjuMandan** [microg/GmsCore#2515]
There's no documentation about the Remote option in DroidGuard operation mode.

Let me guess: if local, the verification is done on same device as microG is running, otherwise is done remotely on another device.

Can we just make a bootloader locked, stock system, innocent device a public DroidGuard runtime server to pass SafetyNet?

**Comment by kaduvert** [comment]
i think it's not that simple

from what i've read, google is very aggressive with its droidguard stuff, and if all of the requests suddenly come from one "phone" because everyone uses it as proxy, then google will block it and all the ones that come after

but it would be good to get more information on how the remote droidguard feature works and if one can use it to bypass play integrity/safetynet

**Comment by lucasmz-dev** [comment]
Apparently in the past droidguard was installed as a separate app (this feels like a really nice improvement to security)  and I believe this is what this was used for

**Comment by ale5000-git** [comment]
@lucasmz-dev
No, the Remote option it is a different things but the "server" part of it probably was never implemented.
